### ISSUE TITLE
Download knitR report example is not working in RCloud

### ISSUE THREAD

**Issue by prateek032** [att/rcloud.shiny#11]
after clicking on Download button, it opens up new tab with the following message

`ERROR: cannot open the connection`

and the browser console message is 

![image](https://cloud.githubusercontent.com/assets/10941328/19995423/20cc86b6-a27d-11e6-866b-161e33351776.png)

**Comment by gordonwoodhull** [comment]
I don't think this is an issue in rcloud.shiny, but somewhere in configuration and permission space. 

The notebook worked when I imported it to my local, once I installed latex. Did not get it working on rcloud-5, although moving `report.Rmd` out of Ganz's home seemed to help somewhat (pandoc errors instead of cannot open errors).
